near/kycdao-ntnft/src/lib.rs

```diff
@@ -297,4 +411,88 @@ mod tests {
 
         contract.mint(123);
     }
+
+    // TODO fix this - contract state is not deleted between tests
+    #[test]
+    #[ignore]
+    fn test_status_modifications() {
```

Comment:
I couldn't find a similar test coverage tool for NEAR, but it might be nice to copy over some of the test cases from EVM and implement them here too.